test/nimble.phx.gen.template/addons/variants/web/sobelow_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule Nimble.Phx.Gen.Template.Addons.Web.SobelowTest do
+  use Nimble.Phx.Gen.Template.AddonCase
+
+  setup %{project: project, test_project_path: test_project_path} do
```

Comment:
If we refactor to tag as I mentioned above, we can apply Credo addon only test case that needed (apply only last one) 💭